packages/create-cloudflare/src/git.ts

```diff
@@ -26,37 +27,11 @@ export const offerGit = async (ctx: C3Context) => {
 		return; // bail early
 	}
 
-	const insideGitRepo = await isInsideGitRepo(ctx.project.path);
-
-	if (insideGitRepo) {
-		// Store that we're in an existing git repo for later reference
-		ctx.gitRepoAlreadyExisted = true;
-
-		// If git flag was explicitly set to false, respect that
-		if (ctx.args.git === false) {
-			return;
-		}
-
-		// If git flag was explicitly set to true, respect that
-		if (ctx.args.git === true) {
-			return;
-		}
-
-		// Otherwise, ask the user if they want to use git
-		ctx.args.git = await processArgument(ctx.args, "git", {
-			type: "confirm",
-			question:
-				"You're in an existing git repository. Do you want to use git for version control?",
-			label: "git",
-			defaultValue: C3_DEFAULTS.git,
-		});
-
-		return;
-	}
-
-	ctx.args.git = await processArgument(ctx.args, "git", {
+	ctx.args.git ??= await processArgument(ctx.args, "git", {
```

Comment:
This simplification means that we only ask the user if `ctx.args.git === undefined`.